docs/source/roadmap.md

```diff
@@ -0,0 +1,287 @@
+# _starfish_ Roadmap
+This document describes the features and timeline for current work on the _starfish_ package.
+
+The purpose of this document is to describe the deliverables that we intend to deliver by June 2019. To accomplish this, we first outline our primary [use cases](#use-cases). With these to guide us, we then define milestones for releases that support these needs [**what we need to build**](#what-we-need-to-build:-starfish-0.1.0), and candidate features which we may consider building in the future [**what we can build later**](candidates-features-for-0.2.0+).
+
+## Use cases
+_starfish_ is being developed to support the data processing and benchmarking needs of the SpaceTx consortium and the Chan Zuckerberg Biohub. We have identified a set of use cases from these groups that _starfish_ will support. Feedback from these users will set the course for future work on _starfish_, beyond the milestones identified here.
+
+### Local processing of data
+Some image-based transcriptomics asays have moderate processing requirements. Users of these assays would like to be able to use _starfish_ to process multiple fields of view of data on their local machines to enable further parameter tuning and spot-checking of results. To support this, we will define a Python API for running _starfish_ on local machines that processes experiments of arbitrary numbers of fields of view, logs, stores data provenance, is multi-processing enabled. Users will also be able to run _starfish_ pipelines through a command-line interface (CLI).
+
+### Parameter selection on single fields of view
+Image-based transcriptomics workflows are highly dependent upon the tissue, organism, and probes being assayed. Researchers typically select image processing parameters by analyzing and optimizing parameter selection in a single field of view. In support of this use case, _starfish_ must enable users to process data for any SpaceTx assay on the local computer of their choosing and interact with visualization tools necessary to evaluate parameter decisions. It must support this functionality for users that are not proficient in Python.
+
+### Scalable processing of SpaceTx assays
+It may not always be possible or time efficient to process an entire experiment on a local machine, therefore _starfish_ must be able to process a complete experiment consisting of hundreds of fields of view that are comprised of up to 10s of terabytes of 2-d TIFF images. For the SpaceTx consortium project, the _starfish_ team agreed to process the data for each of the data contributors, so the _starfish_ team will define a "Scalable Pipeline Runner" that will run _starfish_ pipelines on a parallel computing infrastructure of our choice. We will expose tooling to explore the output formats on a personal computer of our contributor's choosing.
```

Comment:
```suggestion
It may not always be possible or time efficient to process an entire experiment on a local machine. Therefore, _starfish_ must be able to process a complete experiment consisting of hundreds of fields of view that are comprised of up to 10s of terabytes of 2-d images. For the SpaceTx consortium project, the _starfish_ team agreed to process the data for each of the data contributors, so the _starfish_ team will define a "Scalable Pipeline Runner" that will run _starfish_ pipelines on a parallel computing infrastructure of our choice. We will expose tooling to explore the output formats on a personal computer of our contributors' choosing.
```